### PR TITLE
layouts/use-case: add previous and next links

### LINKED ISSUE #56: User stories: link to other stories
on each user story page (e.g., https://relaunch.riot-os.org/use_cases/continental.html), it would be helpful to add links to other user stories, e.g., to navigate to the previous and next user story in the list

### PR DESCRIPTION
**Pull request by leandrolanzieri**
## Contribution description
This PR adds links to the previous and next use cases in the use case single page. Also splits the use cases from the companies that only have their logo on the wall. Additionally, information about how to add elements to these collections is added in the README.md.

Fixes #56 
~~Depends on #60~~

### CODE REVIEW

**Comment by waehlisch** [comment]
regarding the split of supporters and companies, this is confusing. we might have users (e.g., companies) who are not willing to provide more details about the use case(e.g., quote). grouping them under "Supporters" would be incorrect. why did you split? only because of implementation problems of the Next and Previous buttons? ...  i suppose we can fix this by filters (https://jekyllrb.com/docs/liquid/filters/)

File: _layouts/use-case.html [hunk 1]
```diff
@@ -52,6 +72,26 @@ <h4 class="text-uppercase d-inline-block ps-3"><a href="{{ page.company_url | de
           </div>
         </div>
 
+        <div class="col-lg-2 d-flex col-6 order-last align-items-center navigation">
+          {% if page.next.url %}
+          <div class="row ps-2 mb-4 m-lg-0">
+            <div class="col">
+              <a href="{{page.next.url}}" class="h-100 m-auto d-flex align-items-center">
+                <small class="text-end">
+                  <span class="text-muted">Next: </span>
```
Comment (waehlisch) [inline _layouts/use-case.html:81]: "Next" and "Previous" seem redundant to me as we also show ri-arrow... i would rather use "Use Case:" or just show the project title.

rest looks good. it's nice that the boxes move to the bottom if the screen is too small.

**Comment by leandrolanzieri** [comment]
> we might have users (e.g., companies) who are not willing to provide more details about the use case(e.g., quote). grouping them under "Supporters" would be incorrect. why did you split?

Well, supporters is only the collection name and is not reflected on the website, it can be changed. Semantically they are different types of information, the use case requires many fields that the supporter (or logo on the wall) does not need, and they even have an output. Having to add checks or filters everywhere to see if an use case is really an use case or merely a logo made me think that perhaps it's better to split them into two collections.

File: _layouts/use-case.html (hunk 1 above)
Comment (leandrolanzieri) [reply _layouts/use-case.html]: Removed. Added titles to help with accessibility.

**Comment by waehlisch** [comment]
> Well, supporters is only the collection name and is not reflected on the website, it can be changed. Semantically they are different types of information, the use case requires many fields that the supporter (or logo on the wall) does not need, and they even have an output. Having to add checks or filters everywhere to see if an use case is really an use case or merely a logo made me think that perhaps it's better to split them into two collections.

i know. but that some items provide more and others less information, doesn't necessarily qualify for different collections. you can also argue that you show what is provided by someone who uses RIOT, instead of moving between collections.

i also see it from this point: we have a user that provides the logo first, six months later more details are provided, why should we move this item between folders?

**Comment by leandrolanzieri** [comment]
> i also see it from this point: we have a user that provides the logo first, six months later more details are provided, why should we move this item between collections (and folders)?

What about this? : we split all the companies into their own data file. It would look like something like:
```yml
savoir_faire_linux:
  name: Savoir-faire Linux
  logo: logos/savoir-faire-linux-logo.svg
  url: https://savoirfairelinux.com/en

continental:
  name: Continental
  logo: logos/continental-logo.svg
  url: https://www.continental.com/en

locha:
  name: Locha Mesh
  logo: logos/lochamesh-logo.svg
  url: https://locha.io

[...]
```
Then, each use case can reference the company by name:
```yml
project: Chat and send Bitcoin without Internet
quote: In Locha Mesh, we decided to use...
user: Jean Pierre Dudey
user_position: Embedded Software Developer
user_photo: user-photos/Jean-Pierre-Dudey.jpg
company: locha
```

This allows more flexibility, because we don't need to move files around to switch from only the logo to use case. This also allows for potential multiple use cases from the same company without duplicating logos on the wall.

**Comment by leandrolanzieri** [comment]
As a reference, this approach is documented here https://jekyllrb.com/docs/datafiles/#example-accessing-a-specific-author

**Comment by waehlisch** [comment]
sounds good to me @leandrolanzieri

**Comment by leandrolanzieri** [comment]
Done, see #60

**Comment by leandrolanzieri** [comment]
#60 is merged. Rebased on top of current master.

**Review by waehlisch — APPROVED** [review]

**Pull request merged**